.travis.yml

```diff
@@ -1,28 +1,25 @@
+sudo: false
+dist: trusty
+
 language: php
 
 php:
-    - 5.5
     - 5.6
     - 7.0
+    - 7.1
     - hhvm
 
-env:
-    - SYMFONY_VERSION=2.3.*
-    - SYMFONY_VERSION=2.4.*
-    - SYMFONY_VERSION=2.5.*
-    - SYMFONY_VERSION=2.6.*
-    - SYMFONY_VERSION=2.7.*
-    - SYMFONY_VERSION=2.8.*
-    - SYMFONY_VERSION=3.0.*
```

Comment:
AFAIK that BC promise has been broken a few times (think about the form component changes around 2.4-2.6)

I checked other libraries and there seem to be a trend to remove these kind of tests, so you might be right after all.

For reference, here is HTTPlug Bundle's travis config, which is kind of a midway: https://github.com/php-http/HttplugBundle/blob/master/.travis.yml